src/nft/components/explore/CollectionTable.tsx

```diff
@@ -113,7 +115,7 @@ const CollectionTable = ({ data, timePeriod }: { data: CollectionTableColumn[];
           return timePeriod === TimePeriod.AllTime ? (
             <TextCell value="-" />
           ) : (
-            <ChangeCell change={cell.row.original.volume.change} />
+            <ChangeCell change={Math.min(cell.row.original.volume.change, VOLUME_CHANGE_MAX_VALUE)} />
```

Comment:
this logic looks good to me, one thing I'm not sure on though

I think the task wants us to show  "> 9999%" and not just 9999

Reply:
makes sense! already updated the code. Wasn't sure whether to leave or keep a space between `>` and a number, pasted screenshots in the ticket.